packages/datadog-instrumentations/test/passport-http.spec.js

```diff
@@ -95,16 +125,17 @@ withVersions('passport-http', 'passport-http', version => {
 
       expect(res.status).to.equal(200)
       expect(res.data).to.equal('Granted')
-      expect(subscriberStub).to.be.calledOnceWithExactly(
-        {
-          credentials: { type: 'http', username: 'test' },
-          user: { _id: 1, username: 'test', password: '1234', email: 'testuser@ddog.com' }
-        }
-      )
+      expect(subscriberStub).to.be.calledOnceWithExactly({
+        framework: 'passport-basic',
```

Comment:
no, in the passport-http module there are two strategies: `basic` and `digest`. We want to be able to differentiate those two, so I called it passport-basic. Would you prefere something like `passport-http:basic` just in case someone creates a `passport-basic` module ?